### DISCUSSION TITLE
Podman pods support

### DISCUSSION THREAD

**Discussion by rgolangh** [rgolangh/gnome-shell-extension-containers#22] [Ideas]
Today the extension supports only podman containers. Podman supports Pods which are collection of pods on the same 
namespace, and for some scenarios it is really useful. I am guessing this is how podman compose is working.
I can image a typical dev scenario of a podman compse of db+nginx this is super useful that you manage by a click.

I fiddled with gnome drop down menu and I'm really unhappy with the results. I can not get  a nested menu for pods
that will behave the way containers menu do today. The 2nd level of collapsible menu doesn't work.

```
---- Pods menu -----
podX  ->
            |
            stop
            logs
            containers:
                containerA ->
                    stop
                    logs
                    ...
                containerB -|
podY -|
-------------------------
```

I'd be happy to get feedback, thoughts, or more suggestions to go about this.

**Comment by rgolangh** [comment]
related to #38

**Reply by benyaminl** [reply]
Seems Gnome menu still has no sub menu grouping capability? Or we can move it into seperate card, like swipe left, so when clicking the submenu, it will show a card overlay or somekind like that, and there is back button, as a workaround? Or do you think it will broke the flow?

**Reply by rgolangh** [reply]
I guess menus won't go further deep than the 1 level because this will go too big and wide for a menu. It makes sense to me 
that something dynamic that pops by a click and disappears when clicking outside of it should be relatively small and easy to navigate. A window makes more sense for such a tree navigation. 

Thinking of your suggestion, perhaps when clicking on a pod I can swap the whole content of the menu to be Pod specific, and provide "back" button to go back the list of pods and containers.

**Reply by benyaminl** [reply]
Yes, that seems great tbh. haha...
